Sources/App/CRP/SlackCommand+CRP.swift

```diff
@@ -69,3 +68,41 @@ extension SlackCommand {
         })
     }
 }
+
+/// Filter the CHANGELOG entries then orders and format the CHANGELOG text
+///
+/// - Parameters:
+///   - commits: The list of commits gathered between last release and current one
+///   - release: The release for which to build the CHANGELOG text for
+/// - Returns: The text containing the filtered and formatted CHANGELOG, grouped and ordered by jira board
+private func formatChangelog(using commits: [String], for release: GitHubService.Release) -> String {
+    // Only keep SDK commits if release is for SDK
+    let filteredMessages = release.isSDK ? commits.filter { $0.contains("#SDK") || $0.contains("[SDK-") } : commits
+
+    // Group then sort the changes by JIRA boards (unclassified last)
+    let grouped = Dictionary(grouping: filteredMessages) { ticket(from: $0)?.board }
```

Comment:
can we simplify this by doing it like this?

```swift
Dictionary(grouping: filteredMessages) { ticket(from: $0)?.board ?? "ZZZ" }.sorted { $0.key < $1.key }
```

Reply:
Well no because the `ZZZ` is used only for sorting, but later when I build the markdown text from the entries, the title for the ones without any entry is "Other", not "ZZZ"

(If there was a nice way to say "sort by key, but consider `nil` to be greater than anything" I'd use that, just didn't want to complicate the code in the sort closure for so little)